recipe/meta.yaml

```diff
@@ -7,10 +7,10 @@ package:
 
 source:
   url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
-  sha256: 2947f531ad52e81b2436b26608d8198778ac0b4baa7d2925db8b3b3fcb39c8a2
+  sha256: 3024e3dc85ca8b26de33c2c5aec7c7a5f1f4a750463cb1378f1c61b4dc28ec58
 
 build:
-  number: 201
+  number: 0
```

Comment:
So this needs to be `200`. We could also use this trick ( https://github.com/conda-forge/numpy-feedstock/commit/4a98d05bc762321aadb6a385edacb2171d53adad ) to simplify it.

ref: https://conda-forge.org/docs/blas.html